scripts/lib/executionUtils.js

```diff
@@ -22,8 +22,7 @@ function runMain(mainFunction) {
     // callback that will catch both thrown errors and rejected promises
     .then(() => mainFunction())
     .catch((error) => {
-      printError('\nScript exited with error:')
-      printErrorWithCause(error)
```

Comment:
Node 23 now prints error cause by itself, no need to do it ourself else it'll be printed twice